kernel/classes/datatypes/ezuser/ezuser.php

```diff
@@ -27,6 +27,8 @@ class eZUser extends eZPersistentObject
     const PASSWORD_HASH_MYSQL = 4;
     /// Passwords in plaintext, should not be used for real sites
     const PASSWORD_HASH_PLAINTEXT = 5;
+    /// Passwords in bcrypt of password
+    const PASSWORD_BCRYPT = 6;
```

Comment:
Why not name it `PASSWORD_HASH_BCRYPT` to make it consistent with other constants?